test/test_monitor.py

```diff
@@ -98,6 +100,60 @@ def handler(event: Event) -> None:
         log_event(e)
         self.assertEqual(len(events), 2)
 
+TEST_TENSORBOARD = True
+try:
+    from torch.utils.tensorboard import SummaryWriter
+    from tensorboard.backend.event_processing import (
+        plugin_event_multiplexer as event_multiplexer,
+    )
+except ImportError:
+    TEST_TENSORBOARD = False
```

Comment:
can we put this in setUp?

Reply:
this was copied from the tensorboard tests, but done